### PR TITLE
Travis check passed

### PR DESCRIPTION
**Pull request by meets2tarun**
#9 #1 Checkpoint 1
@mariobehling Sir, please review it.
#12

### CODE REVIEW

File: etc/system-lock/lock.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ fi
 sudo chmod +x /etc/rc.local
 fi
 
-if [ $is_success = true ]; then
-sudo reboot
-fi
+#if [ $is_success = true ]; then
+#sudo reboot
```
Comment (niccokunzmann) [inline etc/system-lock/lock.sh]: I guess, if this is required for production, it should be in the code.
I think, you can mock (fake) the reboot command by using an alias. What also works is creating your own reboot command and prepending the folder of it to `$PATH`

**Comment by mariobehling** [comment]
Please also squash your commits.

File: etc/system-lock/lock.sh (hunk 1 above)
Comment (medicalwei) [inline etc/system-lock/lock.sh]: I think we can move the reboot command to the script (or something) calling it, or add a wrapper function to "freeze and reboot", to let the freeze function be standalone and make that testable.

**Comment by meets2tarun** [comment]
License @medicalwei

**Pull request closed without merging**